airflow/example_dags/tutorial.py

```diff
@@ -30,7 +30,7 @@
 from airflow import DAG
 
 # Operators; we need this to operate!
-from airflow.operators.bash import BashOperator
+from airflow.operators.bash_operator import BashOperator
```

Comment:
What is the reason for this change?
`bash_operator` is deprecated
https://github.com/apache/airflow/blob/16f43605f3370f20611ba9e08b568ff8a7cd433d/airflow/operators/bash_operator.py#L26